mechanics/TOC.md

```diff
@@ -74,16 +70,10 @@ committee elections.
 
 ## Vacancies
 
-In the event of a resignation or other loss of an elected TOC member, the
-candidate with the next most votes from the previous election will be offered
-the seat. This process will continue until the seat is filled.
-
-In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. Eligible voters from the most recent election will
-vote in the special election (ie: eligibility will not be redetermined at the
-time of the special election). Any replacement TOC member will serve out the
-remainder of the term for the person they are replacing, regardless of the
-length of that remainder.
+In the event of a resignation or other loss of an elected TOC member, a special
+election for that position will be held as soon as possible. Any replacement TOC
+member will serve out the remainder of the term for the person they are replacing,
+regardless of the length of that remainder.
```

Comment:
This gets me thinking about another alternative. Something like this might be a good middle ground? Changes in _italics_:

> In the event of a resignation or other loss of an elected TOC member, the candidate with the next most votes from the previous _annual_ election will be offered the seat. _If the seat cannot be filled from the  previous annual election, the candidate with the next most votes from the most recent special election will be offered the seat._ This process will continue until the seat is filled. In case this fails to fill the seat, a special election for that position will be held as soon as possible. Eligible voters from the most recent election will vote in the special election (ie: eligibility will not be redetermined at the time of the special election). Any replacement TOC member will serve out the remainder of the term for the person they are replacing, regardless of the length of that remainder.

Reply:
yeah, that would look good to me 👍

Reply:
@julz did you want to update this section and add the section here that @jberkus suggested:
```
"In the event that a TOC seat becomes available with three months or less left until the next regular annual election, the Steering Committee shall appoint a qualified contributor to fill that TOC seat until that election."
```

Reply:
👍 Done. Updated with Dawn and Josh's suggestions